packages/adapters/src/process-instance-adapter.ts

```diff
@@ -58,8 +58,8 @@ IComponent {
         if ("python3" in config.engines) {
             this.logger.trace(gotPython);
             const runnerPath = isTSNode
-                ? "../../../python/runner/runner.py"
-                : "../../python/runner/runner.py";
+                ? "../../python-runner/runner.py"
+                : path.resolve(__dirname, require.resolve("@scramjet/python-runner"));
```

Comment:
in this case it doesn't matter if it isTSNode or not.
i can't make a suggestion here but i should be just 
```const runnerPath = path.resolve(__dirname, require.resolve("@scramjet/python-runner"));```

Reply:
Well, I'd like to treat `isTSNode` as "run directly from source" (as opposed to run from `dist/`). At least that's what this code is intended to do - run python runner from source if the whole thing is ran from source.